runtime/scripts/math.js

```diff
@@ -506,6 +506,26 @@ var math = Numbas.math = /** @lends Numbas.math */ {
         return Math.abs(a-b) <= Math.max( rel_tol * Math.max(Math.abs(a), Math.abs(b)), abs_tol );
     },
 
+    is_scalar_multiple(a,b,rel_tol,abs_tol) {
+      rel_tol = rel_tol === undefined ? 1e-15 : rel_tol;
+      abs_tol = abs_tol === undefined ? 1e-15 : abs_tol;
```

Comment:
`rel_tol` and `abs_tol` aren't used anywhere - they should be passed to `math.isclose`, I suppose?